ci/pipelines/cf-deployment.yml

```diff
@@ -1108,6 +1136,34 @@ jobs:
     - get: cf-deployment-develop
       passed: [ experimental-acquire-pool ]
     - get: relint-envs
+  - task: check-stemcell-versions
```

Comment:
Can we use yaml anchors to avoid duplication?

Reply:
We can but I drop this option base on the fact that 1) we did not use it in general in our repo 2) due to potential readability issues and complexity that comes with yaml anchors.